content/doc/book/managing/plugins.adoc

```diff
@@ -85,6 +85,9 @@ configured Update Center.
 [[install-with-cli]]
```

Comment:
I am not sure how much of this page you are updating, so please ignore my suggestion if it is out of scope, but the first sentence: Most plugins can be installed and used immediately by checking the box adjacent to the blugin and clicking (checking here should be _selecting_) *Install without restart*.